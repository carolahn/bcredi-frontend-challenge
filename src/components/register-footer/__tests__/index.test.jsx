import React from "react";
import renderer from "react-test-renderer";

import RegisterFooter from "../index";

describe("RegisterFooter render test", () => {
  it("renders correctly RegisterFooter", () => {
    const tree = renderer.create(<RegisterFooter />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
