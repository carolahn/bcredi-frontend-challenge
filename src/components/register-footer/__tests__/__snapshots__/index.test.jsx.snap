// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegisterFooter render test renders correctly RegisterFooter 1`] = `
<div
  className="form__footer--min-height"
>
  <div
    className="form__footer"
  >
    <p>
      JÃ¡ fiz o meu cadastro. 
      <a
        href="#entrar"
      >
        Entrar agora.
      </a>
    </p>
  </div>
</div>
`;
